.github/workflows/deploy.yml

```diff
@@ -33,5 +33,11 @@ jobs:
       - name: Login on Remote Machine
         run: ssh "${{ vars.REMOTE_USER }}@${{ vars.REMOTE_DOMAIN }}" -- 'docker login ghcr.io -u "${{ github.actor }}" -p "${{ secrets.GITHUB_TOKEN }}"'
 
-      - name: Start Docker Compose on Remote Machine
-        run: ssh "${{ vars.REMOTE_USER }}@${{ vars.REMOTE_DOMAIN }}" -- 'cd "${{ vars.WORK_DIR }}" && export WORK_DIR="${{ vars.WORK_DIR }}" && export PROD_DOMAIN="${{ vars.REMOTE_DOMAIN }}" && export DEV_DOMAIN="${{ vars.REMOTE_DEV_DOMAIN }}" && docker compose up -d --pull always'
+      - name: Start Docker Compose with profile '${{ inputs.profile }}' on Remote Machine
+        run: |
+            ssh "${{ vars.REMOTE_USER }}@${{ vars.REMOTE_DOMAIN }}" --
+            'cd "${{ vars.WORK_DIR }}" &&
```

Comment:
I don't think this works. It should probably be on the same line. (no newline after `--`)

`&&` should work fine, although I don't think they are needed any more, as `run: |` indicates multiple commands being executed anyway.